proptest-state-machine/src/iterative_runner.rs

```diff
@@ -0,0 +1,861 @@
+use std::{cell::RefCell, marker::PhantomData};
+
+use proptest::{
+    bits::{BitSetLike, VarBitSet},
+    collection::SizeRange,
+    num::sample_uniform_incl,
+    prelude::Just,
+    std_facade::fmt::Debug,
+    strategy::{BoxedStrategy, Strategy, ValueTree},
+    test_runner::{Config, Reason, TestError, TestRunner},
+};
+
+pub fn test_sequential<
```

Comment:
I'm not sure current types would allow to do that easily. The rational to have this function at all for IterativeRunner is to be able to reproduce the failing test. I will rename it to test_iterative as you suggested.